client/src/components/UserProfile/queryGetUserInterests.graphql

```diff
@@ -0,0 +1,9 @@
+query GetUserBio($id: ID!){
```

Comment:
this query is not fetching the bio, you can change the name to reflect what is actually doing